public_dropin_environments/python311_genai_agents/pyproject.toml

```diff
@@ -77,9 +73,10 @@ dependencies = [
     "llama-index-llms-litellm>=0.4.1",
     "llama-index-llms-openai>=0.3.38",
     "opentelemetry-instrumentation-llamaindex>=0.40.5",
-    "pypdf>=6.0.0",
+    "pypdf>=6.1.3",
     "logfire>=4.6.0",
     "pydantic-ai>=1.0.5",
+    "uv>=0.9.6",
```

Comment:
Hm, uv should not in a runtime image, its a dev tool. Lemme check

Reply:
`crewai` depends on it for its `cli` 🤦🏻 https://github.com/search?q=repo%3AcrewAIInc%2FcrewAI%20uv&type=code

Reply:
@az-dr please use this feature of `uv` https://github.com/astral-sh/uv/pull/16528 to exclude `uv` from installing here. @cavitcakir we need the same for `datarobot-genai`.